lib/src/models/item_model.dart

```diff
@@ -0,0 +1,87 @@
+class ItemModel {
+  int _page;
+  List<_Result> _results = [];
+
+  ItemModel.fromJson(Map<String, dynamic> parsedJson) {
+    print(parsedJson['results'].length);
+    _page = parsedJson['page'];
+    List<_Result> temp = [];
+    for (int i = 0; i < parsedJson['results'].length; i++) {
+      _Result result = _Result(parsedJson['results'][i]);
+      temp.add(result);
+    }
+    _results = temp;
+  }
+
+  List<_Result> get results => _results;
+
+  int get page => _page;
+}
+
+class _Result {
```

Comment:
Lets get this model in another class, it would be better 👍

Reply:
Separate class _Result from item_model and renamed to Result